src/interfaces/IWeather.tsx

```diff
@@ -0,0 +1,6 @@
+export type IWeather = {
```

Comment:
I would change the name to just `Weather` and change the file name to `types.ts` (you can add more types to this file in the future) and move it into the src and delete `interfaces`

at the moment the variable, file and folder name don't really describe that is going on in this file